docs/notebooks/analysis/nullpoint.ipynb

```diff
@@ -53,7 +53,8 @@
    "id": "60702e11-905b-43b3-85ab-dc3f85bc9bab",
    "metadata": {},
    "source": [
-    "## How Null Point Finder works\n",
+    "## How the null point finder works\n",
+    "\n",
```

Comment:
I made a few small changes in this notebook like adding blank lines in some Markdown cells.